electron/js/window.js

```diff
@@ -226,6 +226,11 @@ class WindowManager {
 		return { x, y };
 	};
 
+	getWindowId () {
```

Comment:
This isn't currently used by I assume it will be when implementing the _remove from storage when closing window_ part. If it isn't I'll remove it.